proc/proc_darwin.go

```diff
@@ -292,11 +352,25 @@ func (dbp *Process) findExecutable(path string) (*macho.File, error) {
 }
 
 func (dbp *Process) trapWait(pid int) (*Thread, error) {
+trapWaitLoop:
```

Comment:
You don't need this label, the `continue` statements within the `switch` will operate on the `for` loop.